sips/sip-122.md

```diff
@@ -25,38 +25,47 @@ This SIP proposes several improvements to the sUSD shorts  [SIP 103](https://sip
+
+2) The fees earned by minters would increase significantly, as they would be receiving 2.5% of the total value of the liquidatable debt, whereas previously they would only be earning on the fees paid by the liquidator that procures the necessary synths for liquidation.
+
 3) It improves protection against the risk of having under-collateralized loans, by allowing for prompt liquidation regardless of the loan size and currency. This is certainly the case for short positions involving currencies that are not widely available in AMMs. The liquidator would need to either hold stock of the currency or procure it and wait out the price adjustment waiting period before being able to liquidate.
+
 4) It reduces the pressure on the peg, as the liquidator would not need to procure synths from the market in order to trigger a liquidation event.
 
-It is important to mention, that collapsible loans would normally permit further lowering of minimum collateralization ratio, but with the current parameter configuration, there isn't much room to decrease the minimum collateralization ratio further due to the narrow spread between 110% (1 + penalty) and 120% (current minimum collateralization ratio).
+Note that collapsible loans would normally permit further lowering of minimum collateralization ratio, but with the current configuration, there isn't much room to decrease the minimum collateralization ratio further due to the narrow spread between 110% (1 + penalty) and 120% (current minimum collateralization ratio).
+It is important to mention, that liquidation would reward minters who regularly claim as they would receive the value of the  liquidated collateral as well as their proportion of the penalty, while minters who skip on claiming  would see their debt increase, in proportion to their debt pool percentage. The net result being a transfer of value from the liquidated borrower and from minters who don't claim to minters who claim regularly.
 
 
 ### Interaction Delay Circumvention:
 
 Currently multi-collateral loans (including sUSD shorts) have an interaction delay which blocks simultaneous loan interactions as a safety precaution. The interaction delay persists even when the loan is an under-collateralized state, which is problematic because of the following reasons:
-1) The interaction delay could censor borrowers from rescuing their collateral from liquidation via debt repayment, collateral deposit  or loan closing
-2) It prevents the liquidator from being able to settle the borrowers' debt, until the timer expires. Hence, loans could go under-water during times of accelerated persistent decline in prices
+
+1) The interaction delay could censor borrowers from rescuing their collateral from liquidation via debt repayment, collateral deposit  or loan closing.
```

Comment:
Currently the SIP covers skipping 1 hour interaction delay if the loan is being liquidated via `collateral.internalLiquidation()` and `collateral.liquidateFromCollateral()`.  

This would enable borrowers who lock ETH for sUSD to liquidate and close their loans.  

Should these actions bypass the timer if the c-ratio of the short position is below the liquidation ratio ? 

- loan closing (`collateral.closeInternal()`)
- repay the loan (`collateral.repayInternal()`)

This could allow shorts to be borrowed up to the C-ratio limit 120% and circumvent the 1 hour delay and allow them. 

- depositing more collateral (`collateral.depositInternal()`) should be ok as it doesn't close their position.

Reply:
ohhh yeah, I've outlined in the test below that deposit/repayment/closing/liquidation all these should 😅  circumvent the timer when the loan is below 120%... however should doing any of these (even within the same block) lead to c-ratio rising above 120% than the timer is enabled and the waiting period needs to be awaited...

![image](https://user-images.githubusercontent.com/62237347/111267618-75ebc200-862c-11eb-9f76-e116f935c944.png)